docker-compose-generator/docker-fragments/opt-save-memory.yml

```diff
@@ -16,6 +16,11 @@ services:
       BITCOIN_EXTRA_ARGS: |
         dbcache=50
         maxmempool=50
+  dogecoind:
+    environment:
+      DOGECOIN_EXTRA_ARGS: |
```

Comment:
Why does the node support less memory all of a sudden?

Reply:
AFAIK save-memory for dogecoin was intentionally not introduced due to dogecoind needing all the ram

Reply:
DOGE definetely does not need "all the ram" it needs a proper config though to manage it :)